docs/local-development.md

```diff
@@ -17,12 +17,26 @@
    yarn install
    ```
 2. Set up local database using the instructions provided in [database.md](database.md).
-   - Note: You must run `yarn db-import-tests:dev` after setting up your database to import the latest test harness into
+   - Note: You must run `yarn db-import-tests:dev` after setting up your database to import the pinned test harness into
      your project.
 3. Run the server
-   ` yarn dev `
+   `yarn dev`
    Now you can navigate your browser to: [http://localhost:3000/](http://localhost:3000/). You need to use localhost instead of `0.0.0.0` because the cookie needs to be treated as secure.
 
+## Pinned ARIA-AT version
+
+The app imports test plans from the `w3c/aria-at` repository using a pinned commit.
+
+- The pinned SHA lives at `config/aria-at.version`.
+- Local imports without `-c` will read this file (or `ARIA_AT_PINNED_SHA`).
+- CI imports historical commits for coverage and then the pinned commit.
```

Comment:
This is an aside, but it's still unclear to me why exactly we need those historical commits (and now in addition to the pinned commit)... Is this some tech debt that we could also clean up before next year? cc @howard-e